pkg/controller/jenkins/configuration/base/resources/pod.go

```diff
@@ -290,6 +329,23 @@ func newContainers(jenkins *v1alpha2.Jenkins) (containers []corev1.Container) {
 	return
 }
 
+func newInitContainers(jenkins *v1alpha2.Jenkins) (containers []corev1.Container) {
+	initContainer := NewJenkinsMasterContainer(jenkins)
```

Comment:
the initContainer should be of type InitContainer. NewJenkinsMasterContainer returns a Container instead. This would need an addition in types.go in the JenkinsSpec. And the reconcilation would need to be done on top of that. @tomaszsek can you confirm this ?

Reply:
Well, according to https://github.com/kubernetes/api/blob/2a1918b/core/v1/types.go#L2864, InitContainers should be of type []Container